src/modules/dummyqml/DummyQmlViewStep.h

```diff
@@ -19,20 +20,27 @@
 #ifndef DUMMYQMLVIEWSTEP_H
 #define DUMMYQMLVIEWSTEP_H
 
+#include <PluginDllMacro.h>
+#include "locale/TranslatableConfiguration.h"
+#include "utils/CalamaresUtilsSystem.h"
+#include "utils/Variant.h"
```

Comment:
Those two should be in the .cpp

Reply:
Fails to build if I move #include "utils/CalamaresUtilsSystem.h" and #include "utils/Variant.h" from the header to cpp file.